### PR TITLE
[FLINK-34401][docs-zh] Translate "Flame Graphs" page into Chinese

### PR DESCRIPTION
**Pull request by lxliyou001**
## What is the purpose of the change

Translate Flame Graphs to Chinese

## Brief change log

The page is located at "docs/content.zh/docs/ops/debugging/flame_graphs.md"

## Verifying this change

Docs only change.

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): (yes / no) no
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: (yes / no) no
  - The serializers: (yes / no / don't know) no
  - The runtime per-record code paths (performance sensitive): (yes / no / don't know) no
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Kubernetes/Yarn, ZooKeeper: (yes / no / don't know) no
  - The S3 file system connector: (yes / no / don't know) bo

## Documentation

  - Does this pull request introduce a new feature? (yes / no) no
  - If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)

### CODE REVIEW

**Comment by lxliyou001** [comment]
@flinkbot run azure

**Comment by lxliyou001** [comment]
@flinkbot run azure

File: docs/content.zh/docs/ops/debugging/flame_graphs.md [hunk 1]
```diff
@@ -27,36 +27,37 @@ under the License.
...
+- 一个方法的消耗与其他方法相比如何？
+- 堆栈中的一系列调用导致执行特定方法？
 
 {{< img src="/fig/flame_graph_on_cpu.png" class="img-fluid" width="90%" >}}
 {{% center %}}
 Flame Graph
 {{% /center %}}
 
-Flame Graphs are constructed by sampling stack traces a number of times. Each method call is presented by a bar, where the length of the bar is proportional to the number of times it is present in the samples.
+Flame Graphs是通过多次采样堆栈跟踪来构建的。每个方法调用都以一个条形图表示，其中条形图的长度与其在采样中出现的次数成比例。
```
Comment (JingGe) [inline docs/content.zh/docs/ops/debugging/flame_graphs.md:41]:
```suggestion
Flame Graphs是通过多次采样堆栈跟踪来构建的。每个方法调用都由一个条形图表示，其中条形图的长度与其在样本中出现的次数成比例。
```

**Review by JingGe — COMMENTED** [review]
Thanks @lxliyou001 for your contribution. I just left some comments.

**Comment by lxliyou001** [comment]
Hi @JingGe , thank you for your advice, it is my first time participating in the Flink project. i have  added 2 commits with your suggestion, Please review it when you have free time, thank you.

**Comment by lxliyou001** [comment]
> Thanks @lxliyou001 for your contribution. I just left some comments.
Hi @JingGe , thank you for your advice, it is my first time participating in the Flink project. i have added 2 commits with your suggestion, Please review it when you have free time, thank you.

**Comment by lxliyou001** [comment]
Hi @JingGe , thank you for your advice, it is my first time participating in the Flink project. i have added 2 commits with your suggestion, Please review it when you have free time, thank you.

**Comment by lxliyou001** [comment]
@flinkbot run azure

**Comment by lxliyou001** [comment]
@flinkbot run azure

**Comment by lxliyou001** [comment]
@flinkbot run azure

**Comment by lxliyou001** [comment]
@flinkbot run azure

**Comment by lxliyou001** [comment]
@flinkbot run azure

**Comment by JingGe** [comment]
Could you rebase your branch?

**Comment by lxliyou001** [comment]
> Could you rebase your branch?

done, it that OK for you to merge this PR

**Comment by lxliyou001** [comment]
hi @JingGe , I have done with it, is that OK for you to merge this PR? thanks a lot.

**Comment by JingGe** [comment]
> hi @JingGe , I have done with it, is that OK for you to merge this PR? thanks a lot.

sure, let's wait until the CI passed

**Review by Zakelly — COMMENTED** [review]
Hi @lxliyou001 , thanks for your contribution! And would you mind adding space between Chinese and English words, since the [translation specifications](https://cwiki.apache.org/confluence/display/FLINK/Flink+Translation+Specifications) recommended to do so.

File: docs/content.zh/docs/ops/debugging/flame_graphs.md [hunk 2]
```diff
@@ -27,36 +27,37 @@ under the License.
 
 # Flame Graphs
```
Comment (Zakelly) [inline docs/content.zh/docs/ops/debugging/flame_graphs.md:28]: 我建议就直接翻译成火焰图吧，也省去很多空格的烦恼。
```suggestion
# 火焰图 Flame Graphs
```

File: docs/content.zh/docs/ops/debugging/flame_graphs.md [hunk 3]
```diff
@@ -27,36 +27,37 @@ under the License.
...
 {{< img src="/fig/flame_graph_on_cpu.png" class="img-fluid" width="90%" >}}
 {{% center %}}
 Flame Graph
 {{% /center %}}
 
-Flame Graphs are constructed by sampling stack traces a number of times. Each method call is presented by a bar, where the length of the bar is proportional to the number of times it is present in the samples.
+Flame Graphs是通过多次采样堆栈跟踪来构建的。每个方法调用都由一个条形图表示，其中条形图的长度与其在样本中出现的次数成比例。
 
-Starting with Flink 1.13, Flame Graphs are natively supported in Flink. In order to produce a Flame Graph, navigate to the job graph of a running job, select an operator of interest and in the menu to the right click on the Flame Graph tab:  
+从Flink 1.13版本开始，Flink原生支持Flame Graphs。要生成一个Flame Graph，请导航到正在运行的作业的作业图，选择感兴趣的算子，并在右侧菜单中点击“Flame Graph”选项卡：
```
Comment (Zakelly) [inline docs/content.zh/docs/ops/debugging/flame_graphs.md:43]:
```suggestion
从 Flink 1.13 版本开始，Flink 原生支持火焰图。要生成一个火焰图，请导航到正在运行的作业图，选择感兴趣的算子，并在右侧菜单中点击 "Flame Graph" 选项卡： 
```

File: docs/content.zh/docs/ops/debugging/flame_graphs.md [hunk 4]
```diff
@@ -65,26 +66,25 @@ The Off-CPU Flame Graph visualizes blocking calls found in the samples. A distin
...
 {{% center %}}
-Flame Graph in Mixed Mode
+混合模式的 Flame Graph 
 {{% /center %}}
 
-##  Sampling process
+##  采样过程
 
-The collection of stack traces is done purely within the JVM, so only method calls within the Java runtime are visible (no system calls).
+堆栈跟踪的收集纯粹在JVM内部进行，因此只能看到Java运行时内的方法调用（没有系统调用）。
```
Comment (Zakelly) [inline docs/content.zh/docs/ops/debugging/flame_graphs.md:78]:
```suggestion
堆栈跟踪的收集纯粹在 JVM 内部进行，因此只能看到 Java 运行时内的方法调用（看不到系统调用）。
```

File: docs/content.zh/docs/ops/debugging/flame_graphs.md [hunk 5]
```diff
@@ -27,36 +27,37 @@ under the License.
...
-Flame Graphs are constructed by sampling stack traces a number of times. Each method call is presented by a bar, where the length of the bar is proportional to the number of times it is present in the samples.
+Flame Graphs是通过多次采样堆栈跟踪来构建的。每个方法调用都由一个条形图表示，其中条形图的长度与其在样本中出现的次数成比例。
 
-Starting with Flink 1.13, Flame Graphs are natively supported in Flink. In order to produce a Flame Graph, navigate to the job graph of a running job, select an operator of interest and in the menu to the right click on the Flame Graph tab:  
+从Flink 1.13版本开始，Flink原生支持Flame Graphs。要生成一个Flame Graph，请导航到正在运行的作业的作业图，选择感兴趣的算子，并在右侧菜单中点击“Flame Graph”选项卡： 
 
 {{< img src="/fig/flame_graph_operator.png" class="img-fluid" width="90%" >}}
 {{% center %}}
-Operator's On-CPU Flame Graph
+算子级别的 On-CPU Flame Graph
```
Comment (Zakelly) [inline docs/content.zh/docs/ops/debugging/flame_graphs.md:47]:
```suggestion
算子级别的 On-CPU 火焰图
```

File: docs/content.zh/docs/ops/debugging/flame_graphs.md [hunk 6]
```diff
@@ -65,26 +66,25 @@ The Off-CPU Flame Graph visualizes blocking calls found in the samples. A distin
...
-i.e. all [task]({{< ref "docs/concepts/glossary" >}}#task) threads of that operator are sampled in parallel and their stack traces are combined.
-If a method call consumes 100% of the resources in one of the parallel tasks but none in the others,
-the bottleneck might be obscured by being averaged out.
+默认情况下，Flame Graph 的构建是在单个[operator]({{< ref "docs/concepts/glossary" >}}#operator)级别上进行的，
+即该算子的所有[task]({{< ref "docs/concepts/glossary" >}}#task)线程并行采样，并将它们的堆栈跟踪合并起来。
+如果某个方法调用在其中一个并行任务中占用了100%的资源，但在其他任务中没有占用，则可能会被平均化而掩盖住瓶颈。
 
-Starting with Flink 1.17, Flame Graph provides "drill down" visualizations to the task level.
-Select a subtask of interest, and you can see the flame graph of the corresponding subtask.
+从Flink 1.17版本开始，Flame Graph 提供了 "drill down" 可视化到任务级别的功能。
```
Comment (Zakelly) [inline docs/content.zh/docs/ops/debugging/flame_graphs.md:84]:
```suggestion
Flink 从 1.17 版本开始提供了单并发级别火焰图可视化的功能。
```

**Comment by lxliyou001** [comment]
@Zakelly 谢谢建议，还有这个pr没合入之前，每次master变更都要merge，然后merge又要过CI，CI耗时又很长，很是麻烦，我想问问CI在跑的过程中能合入吗

**Comment by Zakelly** [comment]
> @Zakelly 谢谢建议，还有这个pr没合入之前，每次master变更都要merge，然后merge又要过CI，CI耗时又很长，很是麻烦，我想问问CI在跑的过程中能合入吗

如果是文档类的更改，确认不对CI造成影响的可以，committer可以合。master变更的话不用merge回你的更改分支哈，如果有更改可以继续提交commit即可。如果因为有冲突合不进去那需要进行rebase。

**Comment by lxliyou001** [comment]
> > @Zakelly 谢谢建议，还有这个pr没合入之前，每次master变更都要merge，然后merge又要过CI，CI耗时又很长，很是麻烦，我想问问CI在跑的过程中能合入吗
> 
> 如果是文档类的更改，确认不对CI造成影响的可以，committer可以合。master变更的话不用merge回你的更改分支哈，如果有更改可以继续提交commit即可。如果因为有冲突合不进去那需要进行rebase。

好的，非常感谢

File: docs/content.zh/docs/ops/debugging/flame_graphs.md [hunk 7]
```diff
@@ -25,38 +25,39 @@ specific language governing permissions and limitations
...
-# Flame Graphs
+# 火焰图
 
-[Flame Graphs](http://www.brendangregg.com/flamegraphs.html) are a visualization that effectively surfaces answers to questions like:
-- Which methods are currently consuming CPU resources?
-- How does consumption by one method compare to the others?
-- Which series of calls on the stack led to executing a particular method?
+[Flame Graphs](http://www.brendangregg.com/flamegraphs.html) 是一种有效的可视化工具，可以回答以下问题：
+
+- 目前哪些方法正在消耗CPU资源？
```
Comment (Zakelly) [inline docs/content.zh/docs/ops/debugging/flame_graphs.md:32]:
```suggestion
- 目前哪些方法正在消耗 CPU 资源？
```

File: docs/content.zh/docs/ops/debugging/flame_graphs.md [hunk 8]
```diff
@@ -25,38 +25,39 @@ specific language governing permissions and limitations
...
 {{< img src="/fig/flame_graph_on_cpu.png" class="img-fluid" width="90%" >}}
 {{% center %}}
 Flame Graph
 {{% /center %}}
 
-Flame Graphs are constructed by sampling stack traces a number of times. Each method call is presented by a bar, where the length of the bar is proportional to the number of times it is present in the samples.
+火焰图是通过多次采样堆栈跟踪来构建的。每个方法调用都由一个条形图表示，其中条形图的长度与其在样本中出现的次数成比例。
 
-Starting with Flink 1.13, Flame Graphs are natively supported in Flink. In order to produce a Flame Graph, navigate to the job graph of a running job, select an operator of interest and in the menu to the right click on the Flame Graph tab:  
+从 Flink 1.13 版本开始，Flink 原生支持火焰图。要生成一个火焰图，请导航到正在运行的作业图，选择感兴趣的算子，并在右侧菜单中点击"火焰图"选项卡：
```
Comment (Zakelly) [inline docs/content.zh/docs/ops/debugging/flame_graphs.md:43]: 选项卡名字就别翻译了。。。因为flink ui上是英文
```suggestion
从 Flink 1.13 版本开始支持火焰图。要生成一个火焰图，请导航到正在运行的作业图，选择感兴趣的算子，并在右侧菜单中点击 "Flame Graph" 选项卡： 
```

**Review by Zakelly — COMMENTED** [review]
Thank for the update! Overall looks good, only several minor comments.

**Review by Zakelly — APPROVED** [review]
@lxliyou001  Thanks for your effort! LGTM. I'm not sure whether you should rebase and squash all your commits. And @JingGe would you please merge this?

**Comment by lxliyou001** [comment]
@flinkbot run azure

**Comment by lxliyou001** [comment]
@JingGe could you please merge this PR, thanks

**Comment by JingGe** [comment]
@lxliyou001 sure, would you like to squash your commits?

**Comment by lxliyou001** [comment]
@JingGe sure,could you help me do the "Squash and merge" action? thanks.

**Pull request merged**